replica/database.cc

```diff
@@ -1000,6 +1000,13 @@ future<> database::create_local_system_table(
     co_await add_column_family(ks, table, std::move(cfg));
 }
 
+void database::mark_table_ready_for_writes(const schema_ptr& schema) {
```

Comment:
Is this function idempotent? If not, maybe we should do a sanity check that this wasn't done before

Reply:
ok